sdk/core/azure-core/azure/core/pipeline/policies/_base.py

```diff
@@ -61,7 +61,7 @@ class HTTPPolicy(ABC, Generic[HTTPRequestType, HTTPResponseType]):  # type: igno
     """
 
     def __init__(self):
-        self.next = None
+        self.next = None # type: Any
```

Comment:
Being it's an HTTPPolicy, what `next` means and what can be the next type? Is `int` a valid value here?